.gitignore

```diff
@@ -3,5 +3,6 @@
 **/.*.*
 coverage/
 cypress/
+dist/
```

Comment:
Committing dist is not a normal pattern for packages, and will cause large error prone diffs for any updates to the package. 

Since `prepublish` runs the build, we don't need to commit the built `dist/` folder, as it will be available during publishing, and published to NPM registry.

Reply:
This is an artifact from some previous iterations where I wanted to pull in Objective-Redux without publishing to NPM (e.g. from Git directly). It can definitely be removed, now. 👍